client/src/routes/Temple/hooks/useMuteAudioListener.spec.tsx

```diff
@@ -37,14 +41,18 @@ describe('useMuteAudioListener', () => {
       </RecoilRoot>
     );
 
+    mockUseTempleExercise.mockReturnValue({
+      slide: {current: {type: 'reflection'}},
```

Comment:
> it('should toggle audio when state is playing and current slide is not reflection',

should  it not play when reflection? the description doesn't seem to match the assertion 🤔

Reply:
Right you are, changed it